--- gsl-2.1/Makefile.am.orig	2015-12-01 21:29:33.926828670 +0100
+++ gsl-2.1/Makefile.am	2015-12-01 21:30:41.736825828 +0100
@@ -19,7 +19,7 @@
 
 lib_LTLIBRARIES = libgsl.la
 libgsl_la_SOURCES = version.c
-libgsl_la_LIBADD = $(GSL_LIBADD) $(SUBLIBS)
+libgsl_la_LIBADD = $(GSL_LIBADD) $(SUBLIBS) cblas/libgslcblas.la
 libgsl_la_LDFLAGS = $(GSL_LDFLAGS) -version-info $(GSL_LT_VERSION)
 noinst_HEADERS = templates_on.h templates_off.h build.h
 
