--- gsl-0.3f/err/stream.c.glibc21	Thu Mar 11 21:38:54 1999
+++ gsl-0.3f/err/stream.c	Thu Mar 11 21:40:35 1999
@@ -6,7 +6,7 @@
 #include <gsl_errno.h>
 #include <gsl_message.h>
 
-FILE * gsl_stream = stderr ;
+FILE * gsl_stream = NULL;
 gsl_stream_handler_t * gsl_stream_handler = NULL;
 
 void
@@ -33,7 +33,9 @@
 FILE *
 gsl_set_stream (FILE * new_stream)
 {
-  FILE * previous_stream = gsl_stream;
+  FILE * previous_stream;
+  if (gsl_stream == NULL) gsl_stream = stderr;
+  previous_stream = gsl_stream;
   gsl_stream = new_stream;
   return previous_stream;
 }
