--- gsl-0.9.2/histogram/Makefile.am.orig	Sun Aug 19 13:36:29 2001
+++ gsl-0.9.2/histogram/Makefile.am	Sat Sep  8 22:47:56 2001
@@ -10,7 +10,7 @@
 TESTS = test test2d test_resample test2d_resample test_trap test2d_trap
 EXTRA_DIST = urand.c
 
-noinst_HEADERS = urand.c
+noinst_HEADERS = urand.c find.c find2d.c
 
 test_LDADD = libgslhistogram.la ../block/libgslblock.la ../ieee-utils/libgslieeeutils.la ../err/libgslerr.la ../test/libgsltest.la
 test2d_LDADD = libgslhistogram.la ../block/libgslblock.la ../ieee-utils/libgslieeeutils.la ../err/libgslerr.la ../test/libgsltest.la
@@ -30,6 +30,5 @@
 
 libgslhistogram_la_SOURCES = add.c  get.c init.c params.c reset.c file.c pdf.c gsl_histogram.h add2d.c get2d.c init2d.c params2d.c reset2d.c file2d.c pdf2d.c gsl_histogram2d.h calloc_range.c calloc_range2d.c copy.c copy2d.c maxval.c maxval2d.c oper.c oper2d.c
 
-noinst_HEADERS = find.c find2d.c
 
 
